### DISCUSSION TITLE
[Esame di prova 30 dicembre 2023] Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#390] [Esami - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/9709fe8a-c90c-438d-8473-72bbd9712d36)

**Comment by cornflexxx** [comment, marked as answer]
1)
$$L_A\begin{pmatrix}
\begin{vmatrix}
x_1 \\
 x_2\\
 x_2\\
x_4
\end{vmatrix}
\end{pmatrix}=x_1\begin{vmatrix}
2\\
1\\
0\\
-1
\end{vmatrix}+x_2\begin{vmatrix}
5\\
4\\
-3\\
-7
\end{vmatrix}+x_3\begin{vmatrix}
0\\
-1\\
2\\
3
\end{vmatrix}+x_4\begin{vmatrix}
1\\
0\\
1\\
1
\end{vmatrix}=\begin{vmatrix}
2x_1+5x_2+x_4\\
x_1+4x_2-x_3\\
-3x_2+2x_3+x_4\\
-x_4-7x_2+3_x3+x_4
\end{vmatrix}$$

2) 
Trovo un base $\mathcal{B}$ di $Im(L_A)$:  Riduco $A$ ad una matrice trovo la posizione dei pivots che indicano la posizione dei vettori linearmente indipendenti in $A$ , quindi una base di $ImA$.

$$A=\begin{vmatrix}
2 &  5&  0& 1 \\
 1&  4&  -1& 0 \\
 0&-3  & 2 & 1 \\
 -1&  -7&  3& 1
\end{vmatrix}\longrightarrow S=\begin{vmatrix}
-1 &  -7&  3& 1 \\
 0&  -3&  2& 1 \\
 0&0  & 0 & 0 \\
 0&  0&  0& 0
\end{vmatrix}$$

Quindi $A^1,A^2$ sono colonne linearmente indipendenti che costituiscono la base $\mathcal{B}=\\{\underline{v}_1,\underline{v}_2\\}$

$$\underline{v}_1=\begin{vmatrix}
2\\
1\\
0\\
-1
\end{vmatrix},\underline{v}_2=\begin{vmatrix}
5\\
4\\
-3\\
-7
\end{vmatrix}$$

Quindi già sappiamo che $dim(Ker(L_A))=dim\mathbb{R}^4-dim(Im(L_A))=4-2=2$, risolvo allora il sistema omogeneo dato da $S\underline{x}=O$ per trovare la base $\mathcal{C}$ del nucleo.

$$\left\\{ \begin{array}{cl}
x_1=x_3-4x_2 \\
x_2=x_2\\
x_3=x_3\\
x_4=3x_2-2x_3
\end{array} \right.$$

$$\mathcal{C}=\begin{Bmatrix}
\begin{pmatrix}
 -4\\
 1\\
 0\\
3
\end{pmatrix},\begin{pmatrix}
 1\\
 0\\
 1\\
-2
\end{pmatrix}
\end{Bmatrix}$$

3)
Trovo le equazioni cartesiane di $Im(L_A)$, riducendo a scala la matrice formata da $\underline{v}_1,\underline{v}_2$, e trovando le coordinate di $\underline{x}$ che azzerano le $n-rg(Im(L_A))$ righe.

$$Im(L_A)=\begin{vmatrix}
2 & 5\\
 1& 4 \\
 0& -3 \\
 -1& -7
\end{vmatrix}\begin{matrix}
 x_1\\
 x_2\\
x_3 \\
x_4
\end{matrix}\longrightarrow \begin{vmatrix}
0 & 0\\
 0& 0 \\
 0& -3 \\
 -1& -7
\end{vmatrix}\begin{matrix}
 x_1-2x_2-x_3\\
 x_2+x_4-x_3\\
x_3 \\
x_4\end{matrix}$$

**Comment by Elia-Belli** [comment]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/1928e909-47d6-4d8e-b4c5-2130522c2336)

**Comment by Frisbon** [comment]
Non mi viene KerLA, potreste dirmi dove ho sbagliato?
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/909e4e5e-d308-4c82-9054-f2985e1a6a4b)

**Reply by cornflexxx** [reply]
Non hai ridotto bene a scala $A$, ti manca una riga da annullare
